media-api/app/lib/querysyntax/QuerySyntax.scala

```diff
@@ -92,7 +92,7 @@ class QuerySyntax(val input: ParserInput) extends Parser with ImageFields {
 
   def ParentField = rule { capture(AllowedParentFieldName)  ~> resolveNamedField _ }
   def NestedField = rule { capture(AllowedNestedFieldName) ~> resolveNamedField _ }
-  def MatchField = rule { capture(AllowedFieldName) ~> resolveNamedField _ }
+  def MatchField = rule { (capture(AllowedFieldName) | QuotedString | StringWithoutColon) ~> resolveNamedField _ }
```

Comment:
I suspect that `StringWithoutColon` can just be `String` - depending on what should happen with `termA:termB:termC` as discussed on Slack.